tests/jsx-newlines/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -95,21 +95,17 @@ newlines_text = (
 
 newlines_text_spaced = (
   <div>
-
     space above
 
     space below
-
```

Comment:
😉 

I thought about changing that, actually, but figured it wouldn't be worth the diff obfuscation of "this change removed the spaces above and below"